service_identity/src/main/kotlin/org/thechance/service_identity/api/endpoints/walletRoutes.kt

```diff
@@ -19,46 +23,72 @@ fun Route.walletRoute(){
             try {
                 val id = call.parameters["id"]!!
                 val wallet = walletUseCaseContainer.getWalletUseCase.invoke(id).toWalletDto()
```

Comment:
Is there a need for calling invoke here couldn't it just be normal brantheses invocation